src/Webserver/Configuration.php

```diff
@@ -7,8 +7,10 @@
     const DEFAULT_PORT = 8000;
     const DEFAULT_HOST = 'localhost';
     const DEFAULT_DOCROOT = '/web';
+    const DEFAULT_ROUTER = 'index.php';
 
     public function getPort();
     public function getHost();
     public function getDocroot();
+    public function getRouter();
 }
```

Comment:
Similarly maybe you can fix my mistake in missing out the line break in this PR ;-)